app/tests/unit/services/version.spec.js

```diff
@@ -293,3 +296,57 @@ describe('updateIsLatest', () => {
     expect(versionTrx.commit).toHaveBeenCalledTimes(1);
   });
 });
+
+describe('RemoveDuplicateLatest', () => {
```

Comment:
Style: the R in remove doesn't need to be capitalized - we usually retain the same casing as the function we're canonically testing.